cms/settings/dev.py

```diff
@@ -97,5 +97,6 @@
         "0003_customimage__privacy_and_more",
         "0003_customdocument__privacy_and_more",
         "0002_articleseriespage_listing_image_and_more",  # Ignoring NOT NULL constraint on columns
+        "0003_releasecalendarpage_datasets",  # Ignoring NOT NULL constraint on columns
```

Comment:
@MaciekBaron @zerolab Q: how should we be handling null values when adding streamfields columns to rows that already exist in DB?. Since it is stored as JSON, should we be setting default value of `list`?

Reply:
let's do `default=list`

Reply:
In my PR from the other week I explicitly allowed `null` values:

https://github.com/MaciekBaron/dis-wagtail/blob/0c42f95e7f87e10c273f7d9e97c33d9fdf13707b/cms/articles/migrations/0003_statisticalarticlepage_corrections_and_more.py#L14-L18

But frankly I haven't noticed that there was already a migration which was ignoring the NOT NULL constraint. Maybe @zerolab has some strong opinions about this.

Reply:
Done in https://github.com/ONSdigital/dis-wagtail/pull/123/commits/baabb60d37906e2e8e66f380dfe450c3ffb0e4d3